Problem building wheels on Ubuntu for i686

```
        ../src/apytypes_util.h: In function ‘INT_TYPE long_mult(INT_TYPE*, INT_TYPE, INT_TYPE)’:
        ../src/apytypes_util.h:94:18: error: expected unqualified-id before ‘__int128’
           94 |         unsigned __int128 res = (unsigned __int128)src0 * (unsigned __int128)src1;
              |                  ^~~~~~~~
```
Seems like `__int128` is not available here?

Since i686 is 32-bit, maybe it makes more sense to build a 32-bit version? (Which then shouldn't use this code path, unless something else is wrong...)

--- a/.github/workflows/publish-test-pypi.yml
+++ b/.github/workflows/publish-test-pypi.yml
@@ -46,7 +46,7 @@ jobs:
       - uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38  # v5.4.0
 
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.19.0
+        run: python -m pip install cibuildwheel==2.22.0
 
       - name: Build wheels
         run: python -m cibuildwheel --output-dir wheelhouse

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - It is possible to perform arithmetic with Numpy arrays, automatically converting
   the Numpy array to an APyArray.
 - `from_float` now accepts APyTypes scalars.
+- Wheels for Python 3.13.
 
 ### Fixed
 

--- a/src/apytypes_util.h
+++ b/src/apytypes_util.h
@@ -71,38 +71,40 @@ public:
 }
 
 //! Compute multiplication between two apy_limb_t and obtain double length result
-template <typename INT_TYPE>
-[[maybe_unused, nodiscard]] static APY_INLINE std::tuple<INT_TYPE, INT_TYPE>
-long_mult(INT_TYPE src0, INT_TYPE src1)
+[[maybe_unused, nodiscard]] static APY_INLINE std::tuple<apy_limb_t, apy_limb_t>
+long_mult(apy_limb_t src0, apy_limb_t src1)
 {
-    if constexpr (sizeof(INT_TYPE) == 4) {
-        std::uint64_t res = (std::uint64_t)src0 * (std::uint64_t)src1;
-        INT_TYPE high_limb = INT_TYPE(res >> 32);
-        return { high_limb, INT_TYPE(res) };
-    } else {
+#if (COMPILER_LIMB_SIZE == 32)
+    std::uint64_t res = (std::uint64_t)src0 * (std::uint64_t)src1;
+    apy_limb_t high_limb = apy_limb_t(res >> 32);
+    return { high_limb, apy_limb_t(res) };
+#elif (COMPILER_LIMB_SIZE == 64)
 #if defined(__GNUC__)
-        // GNU C-compatible compiler (including Clang and MacOS Xcode)
-        unsigned __int128 res = (unsigned __int128)src0 * (unsigned __int128)src1;
-        INT_TYPE high_limb = INT_TYPE(res >> 64);
-        return { high_limb, INT_TYPE(res) };
+    // GNU C-compatible compiler (including Clang and MacOS Xcode)
+    unsigned __int128 res = (unsigned __int128)src0 * (unsigned __int128)src1;
+    apy_limb_t high_limb = apy_limb_t(res >> 64);
+    return { high_limb, apy_limb_t(res) };
 #elif defined(_MSC_VER)
-        // Microsoft Visual C/C++ compiler
-        INT_TYPE high_limb;
-        INT_TYPE low_limb = INT_TYPE(_umul128(src0, src1, &high_limb));
-        return { high_limb, low_limb };
+    // Microsoft Visual C/C++ compiler
+    apy_limb_t high_limb;
+    apy_limb_t low_limb = apy_limb_t(_umul128(src0, src1, &high_limb));
+    return { high_limb, low_limb };
 #else
-        // No 128-bit multiplication intrinsic found. We could implement this function,
-        // but fail for now so we can clearly see which systems are missing out on these
-        // intrinsics.
-        static_assert(
-            false,
-            "long_mult(INT_TYPE n): No intrinsic available on your compiler. Please "
-            "open an issue at https://github.com/apytypes/apytypes/issues with "
-            "information about the compiler and platform and we will be happy to add "
-            "support for it."
-        );
+    // No 128-bit multiplication intrinsic found. We could implement this function,
+    // but fail for now so we can clearly see which systems are missing out on these
+    // intrinsics.
+    static_assert(
+        false,
+        "long_mult(): No intrinsic available on your compiler. Please "
+        "open an issue at https://github.com/apytypes/apytypes/issues with "
+        "information about the compiler and platform and we will be happy to add "
+        "support for it."
+    );
 #endif
-    }
+#else
+    static_assert(false, "COMPILER_LIMB_SIZE must be 32 or 64");
+    return 0;
+#endif
 }
 
 //! Compute the number of trailing zeros in an integer

